src/Configuration/Config.php

```diff
@@ -18,17 +20,71 @@ class Config
     /** @var PathResolver */
     private $pathResolver;
 
-    public function __construct()
+    /** @var Stopwatch */
+    private $stopwatch;
+
+    /** @var FilesystemCache */
+    private $cache;
+
+    /** @var array */
+    private $timestamps = [];
+
+    public function __construct(Stopwatch $stopwatch)
     {
+        $this->stopwatch = $stopwatch;
+        $this->cache = new FilesystemCache();
         $this->pathResolver = new PathResolver(dirname(dirname(__DIR__)), []);
+        $this->data = $this->getConfig();
```

Comment:
there are 2 places where $this->data is filled, remove it from getConfig()

Reply:
Removed it from `parseConfig()` :-)